trace-gradle-plugin/src/functionalTest/java/io/bitrise/trace/plugin/TraceGradlePluginFunctionalTest.java

```diff
@@ -97,6 +111,8 @@ public void tearDownTest() {
         if (DELETE_TEMP_PROJECT) {
             functionalTestHelper.deleteTemporaryProject(testName);
         }
+
+        FunctionalTestWriter.reset();
```

Comment:
Have to reset, otherwise the console will print with the last set color.

Reply:
interesting - what would the overhead be if we reset after writing each text? I think this should be documented in the javadoc to ensure that if other people use it, we clear it down correctly if it does stay like this.

Reply:
It could be reseted after each line, but it is less effective. Good point on adding this to the javadoc, updated it.

Reply:
Yeah - i think the alternative is explicitly saying each line should be a given colour, and if you don't provide a colour, it should be default. This would have the overhead of setting up the colour and tearing down each time we write with it. Not sure what the overhead would be for this? I'm okay with the current way of working if it's now documented :)

Reply:
The behaviour is captured in the javadoc of the writer.